skeleton/webpack/aurelia_project/tasks/run.ext

```diff
@@ -1,62 +1,8 @@
-// @if feat.babel
-import webpack from 'webpack';
-import Server from 'webpack-dev-server';
-import project from '../aurelia.json';
-import gulp from 'gulp';
-// @endif
-// @if feat.typescript
-import * as webpack from 'webpack';
-import * as Server from 'webpack-dev-server';
-import * as project from '../aurelia.json';
-import * as gulp from 'gulp';
-// @endif
+import { NPM } from 'aurelia-cli/lib/package-managers/npm';
 
-import {config} from './build';
-import configureEnvironment from './environment';
-import {CLIOptions, reportWebpackReadiness} from 'aurelia-cli';
+export default function() {
+  console.log('`au run` is an alias of the `npm start`, you may use either of those; see README for more details.');
 
-function runWebpack(done) {
-  // https://webpack.github.io/docs/webpack-dev-server.html
-  let opts = {
-    host: 'localhost',
-    publicPath: config.output.publicPath,
-    filename: config.output.filename,
-    hot: project.platform.hmr || CLIOptions.hasFlag('hmr'),
-    port: CLIOptions.getFlagValue('port') || project.platform.port,
-    contentBase: config.output.path,
-    historyApiFallback: true,
-    open: project.platform.open || CLIOptions.hasFlag('open'),
-    stats: {
-      colors: require('supports-color')
-    },
-    ...config.devServer
-  }/* @if feat.typescript ** as any/* @endif */;
-
-  // Add the webpack-dev-server client to the webpack entry point
-  // The path for the client to use such as `webpack-dev-server/client?http://${opts.host}:${opts.port}/` is not required
-  // The path used is derived from window.location in the browser and output.publicPath in the webpack.config.
-  if (project.platform.hmr || CLIOptions.hasFlag('hmr')) {
-    config.plugins.push(new webpack.HotModuleReplacementPlugin());
-    config.entry.app.unshift('webpack-dev-server/client', 'webpack/hot/dev-server');
-  } else {
-    // removed "<script src="/webpack-dev-server.js"></script>" from index.ejs in favour of this method
-    config.entry.app.unshift('webpack-dev-server/client');
-  }
-
-  const compiler = webpack(config);
-  let server = new Server(compiler, opts);
-
-  server.listen(opts.port, opts.host, function(err) {
-    if (err) throw err;
-
-    reportWebpackReadiness(opts);
-    done();
-  });
+  const args = process.argv.slice(3);
+  return (new NPM()).install(['--', ...args], process.cwd(), 'start');
```

Comment:
One thing we almost forgot. We hijacked our `npm.install()` api to do other npm scripts, this looks odd to users.

Suggest to improve our `lib/package-managers/base-package-manager.js`
```js

  run(command, args = [], workingDirectory = process.cwd()) {
    return new Promise((resolve, reject) => {
      spawn(
        this.getExecutablePath(workingDirectory),
        [command, ...args],
        { stdio: 'inherit', cwd: workingDirectory }
      )
        .on('close', resolve)
        .on('error', reject);
    });
  }

  // keep existing API so we don't have to touch many files
  install(packages = [], workingDirectory = process.cwd(), command = 'install') {
    return this.run(command, packages, workingDirectory);
  }
```

So we can do `return (new NPM()).run('start', ['--', ...args]);` here.

Another thing to suggest is to re-export NPM and Yarn from `lib/index.js`. So that we can do
```js
import { NPM } from 'aurelia-cli';
```
This will shield us from details of TS conversion, the dist file will be `dist/package-managers/npm`, not in `lib/` folder anymore.

Reply:
done